### DISCUSSION TITLE
乱七八糟的技能

### DISCUSSION THREAD

**Discussion by DZDcyj** [DZDcyj/QSGS-v2-Lua#448] [Ideas]
在这里继续 #394 的讨论

**Comment by DZDcyj** [comment]
@Cactusssssssss 
赌局：xx阶段，你可以选择任意种花色并进行判定，若结果不为你选择的花色之一，你摸x张牌。（x为你选择花色的数量）

北伐：锁定技，当一名角色进入濒死状态时，你摸一张牌；当一名角色死亡时，你摸一张牌并回复一点体力，然后从以下三个技能中获得一个你没有的技能：xx，xx，xx

**Comment by DZDcyj** [comment]
xx：出牌阶段限一次，你可以弃置一张牌，然后本回合内使用与此牌花色相同的基本牌或非延时锦囊牌结算完毕后，视为对相同目标使用无花色的同名牌

**Comment by Cactusssssssss** [comment]
xx：转换技，当你造成伤害时：
①.该伤害+1；
②.该伤害-1。

银狮：锁定技：当你未装备防具时受到超过1点的伤害，你防止多余的伤害；当你失去防具和装备区最后一张牌时，你回复一点体力。

**Reply by DZDcyj** [reply]
转换技【易势】已经在大佬酱实装

**Comment by Cactusssssssss** [comment]
仇富：锁定技，当你使用一张杀后，若目标角色每满足以下一项，则伤害+1；满足不少于两项，则不能响应该杀；满足三项，则非锁定技失效直到回合结束：1.手牌数大于你，2.体力值大于你，3.装备区装备的数量多于你

**Comment by Cactusssssssss** [comment]
玩命：每轮开始时，若你的体力值不大于当前轮次，你可以选择一名色，其与你依次受到对方造成的一点伤害直到一名角色死亡。

**Comment by Cactusssssssss** [comment]
偷袭：当你使用杀指定目标后，你可以令该杀的伤害+1，且目标抵消该杀的方式改为打出一张杀。

**Comment by Cactusssssssss** [comment]
辟谣：当一名角色受到不因卡牌造成的伤害时，你可以判定，若结果为：♠，伤害结算结束后其失去一点体力；♣：你可以更改伤害类型；♦：你防止此伤害，然后其失去等量体力。

**Reply by DZDcyj** [reply]
可以考虑用议事替代判定
